macroslib/src/cpp/mod.rs

```diff
@@ -533,6 +537,23 @@ fn merge_rule(ctx: &mut CppContext, mut rule: TypeMapConvRuleInfo) -> Result<()>
     Ok(())
 }
 
+fn cache_f_code(cached_code: &mut Vec<(SmolStr, String)>, f_module_name: &SmolStr, f_code_str: &str) -> bool {
+    for (module_name, code) in cached_code.iter() {
+        if (module_name == f_module_name) && (code.as_str() == f_code_str) {
+            debug!("module code entry already cached.");
+
+            // Return TRUE as this part of code is already cached.
```

Comment:
Looks like "oil oil", at line before `debug!` tell to reader that this is "already cached case",
and line bellow return true, why also add comment about this?

Reply:
Sorry, I don't understand this: `Looks like "oil oil", at line before debug! tell to reader that this is "already cached case",`.

I'll remove the comment.
Do I keep the `debug!()` or shall I remove it too ?

Reply:
Comment and debug!() removed.

Reply:
"oil oil" I mean that the same idea was repeated in several adjacent lines,
so either of `debug!` or comment was enought.